ios/README.md

```diff
@@ -13,6 +13,8 @@ The source for our published iOS Keyman app is found within the **keyman/** subd
 the views specific to the Keyman app, while the core functionality is found in KeymanEngine.framework.
 
 ### Building
+Building Keyman Web is a precursor for compiling KMEI, so verify your system has all the [Minimum Web Compilation Requirements](../web/README.md#minimum-web-compilation-requirements)
```

Comment:
We need this line in the android README.md too, right?

Reply:
yep, I've already added it in the section

## Keyman for Android Development

Reply:
Sorry, my mistake ...